tests/functional/v2/task/testRepair.js

```diff
@@ -113,16 +119,18 @@ describe('Test Repair', function () {
         await warp10.ingest({ className: 'utapi.repair.event' }, events);
         await repairTask._execute(getTs(-100));
 
-        let results = await warp10.fetch({
-            className: 'utapi.repair.correction', labels: { node: prefix }, start: getTs(1), stop: -1,
-        });
-
-        const series = JSON.parse(results.result[0]);
+        const series = await fetchRecords(
+            warp10,
+            'utapi.repair.correction',
+            { node: prefix },
+            { end: getTs(1), count: -1 },
+            '@utapi/decodeRecord',
+        );
         assert.strictEqual(series.length, 3);
         assertResults(totals, series);
 
         await repairTask._execute(getTs(0));
-        results = await warp10.fetch({
+        const results = await warp10.fetch({
```

Comment:
not possible or no benefit in using `fetchRecords` for this one?

Reply:
I didn't convert it as we're only concerned about the length and don't need to decode, but I'll change it for consistency